docs/core/tools/global-json.md

```diff
@@ -38,7 +38,7 @@ The following example shows the contents of a *global.json* file:
 ```json
 {
   "sdk": {
-    "version": "2.2.100"
+    "version": "3.1.100"
```

Comment:
I'd roll back this change as well. I'm already working on major updates to this page and it would look wrong just change this.

```suggestion
    "version": "2.2.100"
```